custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -33,13 +33,13 @@
           <span class="font-16 font-montserrat font-weight-600 mr-3">
             {$ $ctrl.location_type === 'Awc' ? $ctrl.location_type.toUpperCase() : $ctrl.location_type $}s Ranking
           </span>
-          <img src="{% static "icds_reports/mobile/images/dark-info.png" %}" class="d-inline">
+          {# don't show info bubble until we know what it will do #}
+          {#          <img src="{% static "icds_reports/mobile/images/dark-info.png" %}" class="d-inline">#}
```

Comment:
https://trello.com/c/YJQ0vGpR/232-what-should-clicking-the-i-on-the-rankings-page-do